content/blog/posts/2022-09-30-conference-basics.md

```diff
@@ -0,0 +1,67 @@
+---
+authorId: hannakim
+categories:
+  - devlife
+  - developertips
+  - community
+date: '2022-08-30'
+slug: conference-101
+title: 'Some Takeaways from Navigating My First Tech Conference'
+---
+
+## In-person Conferences Are Back!
+
+I had the chance to attend my first in-person tech conference, [CascadiaJS](https://2022.cascadiajs.com/), earlier this month and I'm very glad to have live events back in full swing again!
+
+As software engineers, we're lucky to be in a field that allowed a fairly easy migration to remote work. Virtual conferences and hackathons were plenty available the last few years, but there is something special about in-person events. They facillitate connection in a way I don't think we can quite replicate digitally, though as metaverse creation tools keep evolving, this may change.
+
+As I’m getting ready to head to my next conference, here are some things I’ll be keeping in mind I wanted to share:
+
+## What to Wear to a Conference
+
+I started my career in software development during the pandemic, so I got very used to living in sweatpants and other forms of (athelesure)[https://www.youtube.com/watch?v=CYRENWT8lz8].
+
+As the conference date approached, I realized I had not figured out what to wear the next few days. Would it be alright to go off media tropes of casually dressed developers, or should I play it safe and get myself some business attire? I asked for advice from more seasoned folks in the field and it turned out people wore a mix of both. One recommendation would be to have some comfortable shoes, since you'll likely spend a decent amount of time walking around the venue or standing around while chatting wtih others.
```

Comment:
```suggestion
As the conference date approached, I realized I had not figured out what to wear the next few days. Would it be alright to go off media tropes of casually dressed developers, or should I play it safe and get myself some business attire? I asked for advice from more seasoned folks in the field and it turned out people wore a mix of both. One recommendation would be to have some comfortable shoes, since you'll likely spend a decent amount of time walking around the venue or standing around while chatting with others.
```